include/HAL/Compile/TargetCompilationManager.h

```diff
@@ -113,9 +117,15 @@ class TargetCompilationManager {
   /// @param name The name of the timing span
   mlir::TimingScope getTimer(llvm::StringRef name);
 
+  /// @brief Emit diagnostics capatured in the Target
+  /// @return Was an error or fatal diagnostic detected
+  bool emitDiagnostics();
```

Comment:
Maybe a Nitpick, would it make more sense to can it `emitTargetDiagnostics` instead?

Reply:
Good point, updated in https://github.com/openqasm/qe-compiler/pull/240/commits/cae7163369ed7d442d8d2aa6b4aee066ee1544dd